test/AlarmClockTest.cpp

```diff
@@ -149,7 +149,7 @@ TEST_F(AlarmClockTest, LongTimeout_ImmediatelyDestructed) {
    unsigned int sec = 1000;
    StopWatch sw;
    std::unique_ptr<AlarmClock<seconds>> acPtr(new AlarmClock<seconds>(sec, FakeSleep));
-   EXPECT_FALSE(acPtr->Expired());
+   EXPECT_EQ(ConvertToMicroSeconds(seconds(sec)), acPtr->SleepTimeUs());
```

Comment:
Great catch @john-gress. Yes, I want to make sure that a long sleep is interrupted with a reset. This will affect the timing of these tests, as the real sleep function is going to sleep for at least 500ms before the Reset has a chance to get to it.